migrations/20230704145116-create_users_table.sql

```diff
@@ -0,0 +1,34 @@
+-- +migrate Up
+
+CREATE TABLE IF NOT EXISTS users (
+    user_id SERIAL PRIMARY KEY,
+    github_username TEXT,
+    github_ID INTEGER,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    deleted_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
```

Comment:
The default `deleted_at` should be null, not current timestamp. Otherwise when records are created, they will be marked as deleted.